highlight_go/main.go

```diff
@@ -63,9 +63,11 @@ func Print(src []byte, w io.Writer, p syntaxhighlight.Printer) error {
 		// TODO: Clean this up.
 		whitespace := string(src[lastOffset:offset])
```

Comment:
Done.

On Fri, Jan 16, 2015 at 11:22 PM, Dmitri Shuralyov <notifications@github.com
> wrote:

> In highlight_go/main.go
> <https://github.com/shurcooL/go/pull/15#discussion-diff-23122723>:
>
> > @@ -63,9 +63,11 @@ func Print(src []byte, w io.Writer, p syntaxhighlight.Printer) error {
> >  		// TODO: Clean this up.
> >  		whitespace := string(src[lastOffset:offset])
>
> Ok, I will. Can you please give me push rights to your fork
> <https://github.com/rocky/go>, so that I can push commits into this PR.
> I'll ask you to review them before merging this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/shurcooL/go/pull/15/files#r23122723>.
>